persistence-cassandra-3.11/src/main/java/io/stargate/db/cassandra/CassandraPersistenceActivator.java

```diff
@@ -106,6 +109,10 @@ private Config makeConfig() throws IOException {
 
     @Override
     public void start(BundleContext context) {
+        ServiceReference<?> metricsReference = context.getServiceReference(Metrics.class.getName());
```

Comment:
Contrary to what I did in other modules, I'm not registering for `serviceChanged` events here. I wonder if that's really necessary for the health-checker module, given that it's always present (downstream modules have an API dependency to it), and it won't be swapped at runtime.

I can adapt this or the other modules to make it uniform.